### PR TITLE
[Gov Ch09] Adoption & Transition Plan

### LINKED ISSUE #680: [Gov Ch08] Recruiting, Engagement, & Adoption
**Scope**
Recruit operators & suppliers  to bring diverse perspective and contributions into the community
Drive membership engagement activities to address community resource demands
Engage industry leaders with Common NFVI value proposition to drive adoption rate
Drive adoption roadmap with VNF Vendors

### PR DESCRIPTION
**Pull request by rabi-abdel**
_This is intended to be kept open for Prague event, more details will be added before discussion in Prague_ 

- For any typos or spelling mistakes, please **fix directly** on PR.
- For any other comments, please provide as review requests.

Fixes #680

### CODE REVIEW

**Comment by hdamker** [comment]
Fine with the approach but exception should be explicitly described including their consequences.

File: doc/gov/chapters/chapter09.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+- **Technology Exceptions** : Using specific technologies that are considered non conformant to CNTT principles (such as PCIe Direct Assignment, exposure of hardware features to VNFs/CNFs).
+- **Version Exceptions**: Using Versions of  Software components, , APIs, or Hardware that are different from the one specified in the specification.
+
+<a name="9.2.3"></a>
+### 9.2.3 Transition Framework
+
+#### VNF/CNF Transition Plan Framework 
+
+Exceptions will be clearly recorded in the Reference Model Appendix which will act as a guidance to VNFs/CNFs vendors of what Exceptions will be allowed in each CNTT release. **Figure 1** below demonstrate the concept.
```
Comment (pgoyal01) [inline doc/gov/chapters/chapter09.md]: The technologies and versions are specified in the RA's. Also, the exceptions have to be RA specific. Hence, suggest replace "Reference Model Appendix" to "appropriate Reference Architecture Appendix."

File: doc/gov/chapters/chapter09.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+<p align="center"><img src="../figures/vnf_cnf_transition.png" alt="Transition" title="VNF/CNF Transition Plan" width="70%"/></p>
+<p align="center"><b>Figure 1:</b> Transition Plan for VNFs/CNFs within CNTT</p>
+
+#### NFVI Transition Plan Framework 
+
+Exceptions will be clearly recorded in Reference Architectures' Appendices which will act as a guidance to NFVI vendors of what Exceptions will be allowed in each CNTT release. **Figure 2** below demonstrate the concept.
+
+- As technology matures, fewer and fewer Exceptions will be allowed in CNTT releases.
+- For each CNTT Release, VNF/CNF can be either:
+  - **Fully Conformant**: Support Target Architecture with all RM Exceptions supported. There should be a technology choice in RA to support RM Exceptions (However, none of the Exceptions allowed in RA has been used).
```
Comment (pgoyal01) [inline doc/gov/chapters/chapter09.md]: Suggest change "Support Target Architecture with all RM Exceptions supported" to "Support the Target Reference Architecture without any exceptions."

File: doc/gov/chapters/chapter09.md (hunk 1 above)
Comment (rabi-abdel) [reply doc/gov/chapters/chapter09.md]: addressed here: https://github.com/cntt-n/CNTT/pull/893/commits/b81cb4050977cf34e8370dc13a29404d96dbe9c4

File: doc/gov/chapters/chapter09.md (hunk 2 above)
Comment (rabi-abdel) [reply doc/gov/chapters/chapter09.md]: fixed in here: https://github.com/cntt-n/CNTT/pull/893/commits/74786114e8ed0a1a3085d00a010e55089540381e

**Comment by rabiabdel** [comment]
No further input received, approving and merging

**Pull request merged**